spark-submit-flamegraph

```diff
@@ -64,7 +64,7 @@ install_deps() {
     fi
   done
 
-  echo -e "[$(date -Ins)] Installing dependencies"
+  echo -e "[$(date)] Installing dependencies"
```

Comment:
Is there variant that works both in Linux and Mac that outputs non localized date string?

Reply:
Here is my Mac output,

```bash
$ date -u
Thu Jan 23 07:09:11 UTC 2020
```

Here is the Debian output,
```
$ uname -a
Linux XYZ 4.9.0-8-amd64 #1 SMP Debian 4.9.144-3.1 (2019-02-19) x86_64 GNU/Linux
$ date -u
Thu Jan 23 07:09:37 UTC 2020
```

Reply:
I think `date +"%FT%T.%s"` is a better replacement than just `date`